homeassistant/components/lock/wink.py

```diff
@@ -20,6 +61,65 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
             add_devices([WinkLockDevice(lock, hass)])
 
 
+    def service_handle(service):
+        """Handler for services."""
+        entity_ids = service.data.get('entity_id')
+        if entity_ids is not None:
+            for lock in hass.data[DOMAIN]['entities']:
+                if lock.entity_id in entity_ids:
+                    if service.service == SERVICE_SET_VACATION_MODE:
+                        lock.set_vacation_mode(service.data.get(ATTR_ENABLED))
+                    elif service.service == SERVICE_SET_ALARM_STATE:
+                        lock.set_alarm_state(service.data.get(ATTR_ENABLED))
+                    elif service.service == SERVICE_SET_BEEPER_STATE:
+                        lock.set_beeper_state(service.data.get(ATTR_ENABLED))
+                    elif service.service == SERVICE_SET_ALARM_MODE:
+                        lock.set_alarm_mode(service.data.get(ATTR_MODE))
+                    elif service.service == SERVICE_SET_ALARM_SENSITIVITY:
+                        lock.set_alarm_sensitivity(service.data.get(ATTR_SENSITIVITY))
+        else:
+            for lock in hass.data[DOMAIN]['entities']:
```

Comment:
If you need to look up things in `hass.data[DOMAIN]['entities']`, why don't you make it a dict? You can still iterate over the values by doing `for x in your_dict.values()`